src/models/symbols.ts

```diff
@@ -45,10 +46,8 @@ function sanitizeFilePath(uri: string) {
     return uri.replace(/^file:\/+/, "");
 }
 
-export function navigateToLocation(symbol: { location: vscode.Location }) {
-    const {uri, range} = symbol.location ?? (symbol as any).corresponding?.location;
-    const line = range.start.line + 1; // zero-base in range.
```

Comment:
Previously only jump to target line, so `charater` is always "0". Now passing the accurate range as `selection`, so previous test case failed. I'll update the case.

Reply:
@jdneo